package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dynamic-query-builder-client",
-  "version": "0.1.2",
+  "version": "0.2.0",
```

Comment:
Why have we jumped to `0.2.0` from `0.1.2` ?

Reply:
we just added a date filter so its a minor change isn't it ?